### PR TITLE
Rahul/ifl 266 mintasset and burnasset do not accept a feerate in requests

### PR DESCRIPTION
**Pull request by patnir**
## Summary

createTransaction and sendTransaction accept an optional feeRate. Adding that option for mintAsset and burnAsset. 

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/routes/wallet/burnAsset.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export interface BurnAssetRequest {
   account: string
   assetId: string
   fee: string
+  feeRate?: string
```
Comment (hughy) [inline ironfish/src/rpc/routes/wallet/burnAsset.ts:15]: I think it makes sense to make `fee` optional here -- if a client passes a fee rate they shouldn't need to also pass a fee

File: ironfish/src/rpc/routes/wallet/mintAsset.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import { getAccount } from './utils'
 export interface MintAssetRequest {
   account: string
   fee: string
+  feeRate?: string
```
Comment (hughy) [inline ironfish/src/rpc/routes/wallet/mintAsset.ts:16]: same re: optional fee

File: ironfish/src/rpc/routes/wallet/burnAsset.ts (hunk 1 above)
Comment (patnir) [reply ironfish/src/rpc/routes/wallet/burnAsset.ts]: donie

File: ironfish/src/rpc/routes/wallet/mintAsset.ts (hunk 2 above)
Comment (patnir) [reply ironfish/src/rpc/routes/wallet/mintAsset.ts]: done

**Review by hughy — APPROVED** [review]

**Pull request merged**